tutorials/Atmos/agnesi_hs_lin.jl

```diff
@@ -0,0 +1,306 @@
+# model. The purpose of this is to populate the
+# [`ClimateMachine.AtmosLESConfiguration`](@ref LESConfig) with arguments
+# appropriate to the problem being considered.
+function config_agnesi_hs_lin(FT, N, resolution, xmax, ymax, zmax)
+
+    u_relaxation = SVector(FT(20), FT(0), FT(0))
+    c_sponge = 1
+    amp = 1.0
+
+    # Rayleigh damping
+    zsponge = FT(12500.0)
+    rayleigh_sponge =
+        RayleighSponge{FT}(zmax, zsponge, c_sponge, u_relaxation, 2)
+
+    ##SR LSRK144
+    ode_solver = ClimateMachine.ExplicitSolverType(
+        solver_method = LSRK144NiegemannDiehlBusch,
+    )
+
+    source = (Gravity(), rayleigh_sponge)
+
+    temp_profile_ref = DecayingTemperatureProfile{FT}(param_set)
+    ref_state = HydrostaticState(temp_profile_ref)
+    nothing # hide
+
+    _C_smag = FT(0.23)
+    model = AtmosModel{FT}(
+        AtmosLESConfigType,
+        param_set;
+        turbulence = SmagorinskyLilly(_C_smag),
+        moisture = DryModel(),
+        source = source,
+        tracers = NoTracers(),
+        init_state_conservative = init_agnesi_hs_lin!,
+        ref_state = ref_state,
+    )
+
+    config = ClimateMachine.AtmosLESConfiguration(
+        "Agnesi_HS_LINEAR",      # Problem title [String]
+        N,                       # Polynomial order [Int]
+        resolution,              # (Δx, Δy, Δz) effective resolution [m]
+        xmax,                    # Domain maximum size [m]
+        ymax,                    # Domain maximum size [m]
+        zmax,                    # Domain maximum size [m]
+        param_set,               # Parameter set.
+        init_agnesi_hs_lin!,     # Function specifying initial condition
+        solver_type = ode_solver,# Time-integrator type
+        model = model,           # Model type
+        meshwarp = setmax(warp_agnesi, xmax, ymax, zmax),
+    )
+
+    return config
+end
+
+function main()
+
+    FT = Float64
+
+    N = 4
+    Δh = FT(1200)
+    Δv = FT(240)
+    resolution = (Δh, Δh, Δv)
+    xmax = FT(240000)
+    ymax = FT(4800)
+    zmax = FT(30000)
+    t0 = FT(0)
+    hrs = FT(1)
+    timeend = FT(hrs * 60 * 60)
+
+    Courant = FT(0.25)
+
+    # Assign configurations so they can be passed to the `invoke!` function
+    driver_config = config_agnesi_hs_lin(FT, N, resolution, xmax, ymax, zmax)
+    solver_config = ClimateMachine.SolverConfiguration(
+        t0,
+        timeend,
+        driver_config,
+        init_on_cpu = true,
+        Courant_number = Courant,
+    )
+
+    # User defined filter (TMAR positivity preserving filter)
+    cbtmarfilter = GenericCallbacks.EveryXSimulationSteps(1) do (init = false)
+        Filters.apply!(solver_config.Q, 6, solver_config.dg.grid, TMARFilter())
+        nothing
+    end
+
+    # Invoke solver (calls `solve!` function for time-integrator), pass the driver, solver and diagnostic config
+    # information.
+    result = ClimateMachine.invoke!(
+        solver_config;
+        #user_callbacks = (cbtmarfilter,),
```

Comment:
Was this intentional? Just making sure the filter was not meant to be applied.

Reply:
yes, no need for filter here. It doesn't hurt to have it on but it should not be necessary.